package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -344,7 +344,10 @@ cdef class DCDFile:
         cdef int with_unitcell = 1
 
         if isinstance(remarks, six.string_types):
-            remarks = bytearray(remarks, 'ascii')
+            try:
+                remarks = bytearray(remarks, 'ascii')
+            except UnicodeDecodeError:
+                remarks = bytearray(remarks)
```

Comment:
shouldn't we always try to write ascii encoded remarks? I guess this is what most other programs would expect.